SR2023/2023-03-15-competition-details..md

```diff
@@ -0,0 +1,66 @@
+
+Before then, we have a few important updates and reminders for you:
+
+## Media Consent
+
+Throughout the competition, Student Robotics volunteers will be taking photos and videos for the purpose of prompting and marketing the charity. The arena will also be [livestreamed](https://studentrobotics.org/events/sr2023/competition/#livestream) on our YouTube channel. If this is a problem for you or your competitors, please let our reception desk know when you arrive.
+
+The competition this year is a public event, and we do not have exclusive access to the venue.
+
+## Equipment
+
+Please ensure that your team bring at least one laptop or appropriate device with them to the competition to access the docs and work on your robot's code. We are unable to provide any computing devices for competitor use at the competition. We will be providing a wireless internet connection to all teams.
+
+Any tools that your team requires to modify their robot must be brought to the competition by the team: we cannot provide any. In particular, the use of any dangerous tools must be supervised by the responsible adult for the team. Power tools may only be used in the designated "Power Tools" area.
+
+You must bring all of the robotics kit (including the white box it came in) to the competition, so that we can collect it at the end of the event.
+
+Teams will be allocated an area of the venue to work in, and a table to work on (known as your "pit"). You will have plenty of chance to test your robot in the full arena during _Tinker Time_.
+
+## Robot design considerations
+
+At the competition, all robots undergo a safety check. This not only confirms that the robot satisfies our safety regulations, but also the rules. Of note are a particular few we often see teams forgetting until the last minute:
+
+- Please make sure the power button is obvious and easily accessible. In the event of an issue in the arena, we don't want to be hunting for the switch as we try and quickly turn off your robot.
+- Ensure your battery is protected, ideally housed in its own rigid container.
+- Make sure you have a way to mount the flag. We will have a few spare mounts, but please do not rely on this. [Plastic push-fit couplers](https://www.screwfix.com/p/jg-speedfit-plastic-push-fit-equal-tees-15mm-5-pack/74470) work well for this.
+- Make sure your robot fits inside the size constraints. If it doesn't, it will have a meeting with the _saw of shame_.
+- Make sure there are 2 spare USB ports: 1 for your code, and 1 for our "Competition USB". This Competition USB tells your robot that it's in the competition and which corner it's starting in (and thus populates the `R.zone` property). You will not be able to compete without this.
+
+All of these and more can be found in our [safety regulations](https://studentrobotics.org/docs/kit/safety-regulations) and [rules](https://studentrobotics.org/rules).
+
+## Attendance details
```

Comment:
Since we're asking them to do something here, I wonder if we should lead with this section? (and then repeat the CTA after the last section too?)